tests/ipython/test_ipython.py

```diff
@@ -266,3 +245,23 @@ def test_line_magic_with_invalid_arguments(self, mocker, ipython):
             UsageError, match=r"unrecognized arguments: --invalid_arg=dummy"
         ):
             ipython.magic("reload_kedro --invalid_arg=dummy")
+
+
+class TestProjectPathResolution:
+    def test_only_path_specified(self):
+        result = _resolve_project_path(path="/test")
+        expected = Path("/test").resolve()
+        assert result == expected
+
+    def test_only_local_namespace_specified(self):
+        result = _resolve_project_path(
+            local_ns={"project_path": Path("/test").resolve()}
+        )
+        expected = Path("/test").resolve()
+        assert result == expected
+
+    def no_path_no_local_namespace_specified(self, mocker):
```

Comment:
should it be a test?
```suggestion
    def test_no_path_no_local_namespace_specified(self, mocker):
```

Reply:
Nice, thanks! Fixed in [211706c](https://github.com/kedro-org/kedro/pull/1955/commits/211706ced9422f0a56e77e7fc0cee8551b067a6c)